Kudu.Services/ServiceHookHandlers/GithubHandler.cs

```diff
@@ -17,6 +36,25 @@ public override DeployAction TryParseDeploymentInfo(HttpRequestBase request, JOb
             return DeployAction.UnknownPayload;
         }
 
+        protected override string determineSecurityProtocol(JObject repository)
+        {
+            string repositoryUrl = repository.Value<string>("url");
+            bool isPrivate = repository.Value<bool>("private");
+            if (isPrivate)
+            {
+                repositoryUrl = repository.Value<string>("ssh_url");
+            }
+            else if (!String.Equals(new Uri(repositoryUrl).Host, "github.com"))
```

Comment:
`StringComparison.OrdinalIgnoreCase`. I'd expect a code analysis error without it. Try running build.cmd from command line (at repo root).